atproto/firehose/client.py

```diff
@@ -176,8 +149,22 @@ def _process_message_frame(self, frame: 'MessageFrame') -> None:
             else:
                 traceback.print_exc()
 
-    def start(self, *args, **kwargs):
-        super().start(*args, **kwargs)
+    def start(
```

Comment:
let's rework `time.sleep(self._get_reconnection_delay())` here the same as in the async version? `...threading.Event().wait(self._get_reconnection_delay()) ...`

Reply:
(see above comment)